src/pyclarify/models/data.py

```diff
@@ -157,18 +162,15 @@ class SignalInfo(BaseModel):
     type: TypeSignal = TypeSignal.numeric
     description: str = ""
     labels: Dict[LabelsKey, List[str]] = {}
```

Comment:
Possible to use this?
```suggestion
    labels: Dict[LabelsKey, Set[str]] = {}
```

Reply:
Using set changes the labels from:
```python
labels={'depth': [], 'locality': ['Langøy'], 'site': ['Langøy'], 'topic': ['efb/LA/FL/AT/1'], 'unit-type': ['Flåte'], 'unit_name': ['Flåte']}
```
to
```python
labels={'depth': set(), 'locality': {'Langøy'}, 'site': {'Langøy'}, 'topic': {'efb/LA/FL/AT/1'}, 'unit-type': {'Flåte'}, 'unit_name': {'Flåte'}}
```

I belive that the current version is inline with the documentation.

Reply:
Array works.
Just be aware that we _treat_ it as a set on the server side, which means that if you submit `["foo", "foo", "bar"]`, we store `["bar", "foo"]`